.github/workflows/test-all.yml

```diff
@@ -91,6 +91,11 @@ jobs:
       run: echo "FIREBASE_CI_TEST_START_TIME=$(date +%s)" >> $GITHUB_ENV
     - name: Run unit tests
       # Ignore auth and firestore since they're handled in their own separate jobs.
+      # TODO (dlarocque): Fix flakes in installations, database, database-compat, storage-compat,
```

Comment:
Isn't this workflow file all Chrome? Also isn't the skipping logic in testConfig.ts?

Reply:
Yeah this comment shouldn't be here- and there's already a comment in the test config so I'll just remove this.